sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/Aggregation.java

```diff
@@ -28,17 +28,15 @@
  */
 @Evolving
 public final class Aggregation implements Serializable {
-  private AggregateFunc[] aggregateExpressions;
-  private FieldReference[] groupByColumns;
+  private final AggregateFunc[] aggregateExpressions;
+  private final FieldReference[] groupByColumns;
 
   public Aggregation(AggregateFunc[] aggregateExpressions, FieldReference[] groupByColumns) {
     this.aggregateExpressions = aggregateExpressions;
     this.groupByColumns = groupByColumns;
   }
 
-  public AggregateFunc[] aggregateExpressions() {
-    return aggregateExpressions;
-  }
+  public AggregateFunc[] aggregateExpressions() { return aggregateExpressions; }
```

Comment:
why do we only change the code style of this methd?

Reply:
Sorry, the method was automatically collapsed into one single line in my IDE, I only saw one line there. I tried to be careful, but still missed this one.